Add chkdsk at troubleshooting.md (#165)
Microsoft-community/Community-website: wiki/troubleshooting.md

For more information about SFC, check [this page](https://support.microsoft.com/en-us/help/929833/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system)

## CHKDSK

> This utility checks the file system and file system metadata of a volume for errors.

Reviewer: ```suggestion
This utility checks the file system and file system metadata of a volume for errors.

```

Author: It's identical

Reviewer: Line break

Author: Oh. Ok

Author: I can't find how to acces to the .md to edit it, because GitHub is saying that I can't replace it with an identical text.

> To use CHKDSK, open CMD (aka. Command Prompt) with administrator privileges and type `<volume> /f`.

Reviewer: ```suggestion
To use CHKDSK, open the Command Prompt with Administrator Privileges and type `chkdsk <volume> /f`.
```

Reviewer 2: There's no need to capitalize "Administrator Privileges," and "the" isn't strictly necessary.
```suggestion
To use CHKDSK, open Command Prompt with administrator privileges and type `chkdsk <volume> /f`.
```
Alternatively:
```suggestion
To use CHKDSK, open Command Prompt (be sure to select 'run as administrator') and type `chkdsk <volume> /f`.
```
Either of these is fine.

> **Replace `<volume>` with the actual volume you want to scan, such as `C:`.** Press enter to run the command.

Reviewer: ```suggestion
**Replace `<volume>` with the actual volume you want to scan, such as `C:`,** or type `chkdsk /f` to check your C: drive and press <kbd>Enter</kbd> to run the command.

```

> If you are checking drive C:, you may get an warning that CHKDSK cannot lock the partition and the computer needs to restart to continue. To confirm, type `Y` and press enter. CHKDSK will then check the drive the next time you restart your computer.

Reviewer: ```suggestion
If you are checking drive C:, you may get a warning that CHKDSK cannot lock the partition and the computer needs to restart to continue. To confirm, type `Y` and press enter. CHKDSK will then check the drive the next time you restart your computer. 
::: tip Note
If you do not wish to restart your computer, you can run the command `chkdsk` which will check your drive in read-only mode.
:::
```

> For more information about CHKDISK, check [this page](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk?tabs=event-viewer)

Reviewer: ```suggestion
For more information about CHKDISK, visit [Microsoft's Documentations](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk).
```

Reviewer 2: I am going to be pedantic and say the link text should be `Microsoft's documentation`, not `Microsoft's Documentations`
